src/springfield/fields.py

```diff
@@ -339,6 +340,20 @@ class UrlField(StringField):
     """
     :class:`Field` with a URL value
     """
+    def adapt(self, value):
+        """
+        Validate that the `value` has a valid URL format containing
+        a scheme and network location using urlparse.
+
+        Supported schemes include `http`, `https` and `ftp`.
```

Comment:
Do we want to restrict schemes? I suppose that's the difference between an URI and URI but there are a lot more schemes possible.